exchanges/kucoin/kucoin_types.go

```diff
@@ -763,12 +763,12 @@ type StopOrder struct {
 
 // AccountInfo represents account information
 type AccountInfo struct {
-	ID          string       `json:"id"`
-	Currency    string       `json:"currency"`
-	AccountType string       `json:"type"` // Account type:，main、trade、trade_hf、margin
-	Balance     types.Number `json:"balance"`
-	Available   types.Number `json:"available"`
-	Holds       types.Number `json:"holds"`
+	ID          string        `json:"id"`
+	Currency    currency.Code `json:"currency"`
+	AccountType string        `json:"type"` // Account type:，main、trade、trade_hf、margin
```

Comment:
```suggestion
	AccountType string        `json:"type"` // Account type: main, trade, trade_hf, margin
```

Reply:
WTF was that 😳 

Fixed https://github.com/gbjk/gocryptotrader/commit/5420328bd